dvc/repo/get.py

```diff
@@ -50,11 +53,6 @@ def get(url, path, out=None, rev=None):
     tmp_dir = os.path.join(dpath, "." + str(shortuuid.uuid()))
     try:
         with external_repo(cache_dir=tmp_dir, url=url, rev=rev) as repo:
-            # Note: we need to replace state, because in case of getting DVC
-            # dependency on CIFS or NFS filesystems, sqlite-based state
-            # will be unable to obtain lock
-            repo.state = StateNoop()
-
```

Comment:
Don't delete this, we will go back to zc.lockfile soon and won't be configuring flufl lock for this.

Reply:
We will still need to use sqlite nolock=1, so this won't be need.

Reply:
Forgot about it. Thanks for reminding 👍